src/pages/Login.tsx

```
@@ -1,22 +1,62 @@
 import { Button } from '../components/Button'
 import { AuthLayout } from '../components/AuthLayout'
+import { useNavigate } from 'react-router-dom'
+import { login } from '../apis/auth'
+import { LoginFormType } from '../apis/user/type'
 
 function Login() {
+  const navigate = useNavigate()
+  const [form, setForm] = useState<LoginFormType>({ mail: '', password: '' })
+  const [errors, setErrors] = useState<LoginFormType>({ mail: '', password: '' })
+
+  const mailRegExp = /^[a-zA-Z0-9+-\_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/
```

ESLint

[error] Unnecessary escape character: \_.

(no-useless-escape)